CairoMakie/src/display.jl

```diff
@@ -133,7 +127,7 @@ end
 
 Converts anything like `"png", :png, "image/png", MIME"image/png"()` to `"image/png"`.
 """
-function to_mime_string(mime::Union{String, Symbol, MIME})
+function to_mime_string(mime::Union{String,Symbol,MIME})
```

Comment:
I'd prefer this and also default args like `get_all_plots(scene, plots=AbstractPlot[])` to keep the spaces. Helps with readability and produces nicer line breaks in github diffs imo.

Reply:
Thanks, I've set `whitespace_in_kwargs = true` and `whitespace_typedefs = true`.